lib/Mojo/Date.pm

```diff
@@ -38,6 +47,29 @@ sub parse {
     $offset = (($8 * 3600) + ($9 * 60)) * ($7 eq '+' ? -1 : 1) if $7;
   }
 
+  # ISO 8601 (2000-01-02 03:04:05.678+0900)
+  # Groups: 2000,-,01,02,undef,undef,undef,03,:,04,05.678,+,09,00
+  elsif ($date =~ $ISO8601_RE) {
+    ($year, $h, $m, $s) = ($1, $8//0, $10//0, $11//0);
+    if ( $day = $7 or my $week = $5 ) {
+      $month = 0;
+      unless($day) {
+        my $days = 0;
+        $days += Time::Local::_is_leap_year($_)?366:365 for (1970..($year - 1));
```

Comment:
The function `_is_leap_year` does not appear to be part of the public Time::Local API.

Reply:
It is a private method of Time::Local. It is very simple so it can be copied to Mojo::Date if possible.